debian/rules

```diff
@@ -49,7 +61,7 @@ endif
 endif
 
 override_dh_gencontrol:
-	echo "extra:Depends=$(APT_PKG_DEPS) $(DISTRO_INFO_DEPS)" >> debian/ubuntu-pro-client.substvars
+	echo "extra:Depends=$(APT_PKG_DEPS) $(DISTRO_INFO_DEPS) ${PYTHON3_PKG_RESOURCES}" >> debian/ubuntu-pro-client.substvars
```

Comment:
Is there a reason we're using curly braces for this one when we use parentheses for the other similar variables?

Reply:
Good point, they are now consistent